iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/mqtt5/Mqtt5IotHubConnection.java

```diff
@@ -267,6 +276,9 @@ else if (proxySettings.getProxy().type() == Proxy.Type.HTTP)
                 throw transportException;
             }
 
+            this.mqtt5Client.setMqttAsyncClient(this.mqttAsyncClient);
+            this.mqtt5Client.start();
```

Comment:
Within `open()`, pass the `mqttAsyncClient` into the `mqtt5Client` initialized above.